vfio-ubind/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# vim swap files
```

Comment:
might wanna get this shit out of the way one day (not lang-dependent so I moved it to the root gitignore)